packages/zoe/src/zoeService/zoe.js

```diff
@@ -229,15 +240,19 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
         /** @type {InstanceAdmin} */
         return {
           addZoeSeatAdmin: zoeSeatAdmin => zoeSeatAdmins.add(zoeSeatAdmin),
-          tellZCFToMakeSeat: (
+          tellZCFOfferAdded: (
             invitationHandle,
             zoeSeatAdmin,
-            seatData,
-            seatHandle,
+            proposal,
+            initialAllocation,
           ) => {
-            return E(
-              /** @type {Promise<AddSeatObj>} */ (addSeatObjPromiseKit.promise),
-            ).addSeat(invitationHandle, zoeSeatAdmin, seatData, seatHandle);
+            zoeSeatAdmin.setProposal(proposal);
+            return E(offerUpdaterPromiseKit.promise).offerAdded(
+              invitationHandle,
+              zoeSeatAdmin,
```

Comment:
`zoeSeatAdmin` is being passed to `zcf`, but it has a method `setProposal` which should never be used from the `zcf` side. I think we need to remove that method.